include/oboe/AudioStream.h

```diff
@@ -28,6 +28,12 @@
 
 namespace oboe {
 
+/**
+ * The default number of nanoseconds to wait for when performing state change operations on the
```

Comment:
Do we want this?

Reply:
I think it's good to have a sensible default value for timeouts. Did you want to remove it?